Sources/FoundationInternationalization/CMakeLists.txt

```diff
@@ -54,3 +54,5 @@ set_target_properties(FoundationInternationalization PROPERTIES
 
 set_property(GLOBAL APPEND PROPERTY SWIFT_FOUNDATION_EXPORTS FoundationInternationalization)
 _swift_foundation_install_target(FoundationInternationalization)
+install(FILES $<TARGET_LINKER_FILE:FoundationInternationalization>
```

Comment:
nit: should we just add this to `_swift_foundation_install_target` so that this is automatically applied for any targets we might install in the future?

Reply:
Sure, let me sink it into that.